lib/assignDisabledRanges.js

```diff
@@ -126,6 +126,13 @@ module.exports = function (root, result) {
 		return comment.inline || comment.raws.inline;
 	}
 
+	/**
+	 * @param {PostcssComment} comment
+	 */
+	function isStylelintCommand(comment) {
+		return /^stylelint-(disable|enable)/.test(comment.text);
```

Comment:
I just remembered that we have `disableCommand` and `enableCommand` in this file. I think we should use them instead of hardcoded values. What do you think?

Reply:
Yeah, that makes sense. Should I just call `startsWith` with both or should I put a compiled RegExp in a global variable and use that? (since a regex literal couldn't be used in this case)

Reply:
I would choose `startsWith` because it's easier to understand.